### DISCUSSION TITLE
error eslint when I create a new project next

### DISCUSSION THREAD

**Discussion by SandroFrazaoS** [vercel/next.js#30000] [Help] [answered]
Hello
I'm having eslint issues when I run the command yarn create next-app, the error below occurs
![erro](https://user-images.githubusercontent.com/54441280/137643989-01759bfe-50e8-4736-be8c-b19d0b15e635.png)

how can i resolve this error?

**Comment by SandroFrazaoS** [comment]
to solve the above error I had to update nodejs, after the update there was no error.
![Screenshot_1](https://user-images.githubusercontent.com/54441280/138002697-ddefa1e0-6677-4896-8a5b-70010a95339f.png)

but when i run the command npm run dev, an error occurs and I can't run the next
![Screenshot_14](https://user-images.githubusercontent.com/54441280/138005461-c8692124-b1ef-4425-a51f-fbfd80da169c.png)

**Comment by SandroFrazaoS** [comment, marked as answer]
resolved after changing node version from 17.0.1 current to 14.18.1 LTS

**Reply by samsharma199** [reply]
How to change version from 17.0.1 to 14.18.1 LTS

**Reply by SandroFrazaoS** [reply]
Hello,
I uninstalled nodejs and installed it again but in a lower version.
